<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>831</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView {
	background-color: #000000;
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textDisplay">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>501</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="easyBut">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 5px solid #00FF00;
	font: 18pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #B8FFC4;
	border: 5px solid #B8FFC4;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>EASY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mediumBut">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 5px solid #00FF00;
	font: 18pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #B8FFC4;
	border: 5px solid #B8FFC4;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>MEDIUM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hardBut">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 5px solid #00FF00;
	font: 18pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #B8FFC4;
	border: 5px solid #B8FFC4;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>HARD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitBut">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cBut">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dBut">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eBut">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gBut">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kBut">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lBut">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jBut">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>J</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fBut">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bBut">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iBut">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBut">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hBut">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mBut">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oBut">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qBut">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vBut">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tBut">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wBut">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rBut">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uBut">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>U</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yBut">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zBut">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sBut">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xBut">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textHint">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier New'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textHangman">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>139</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	background-color: #000000;
	color: #00FF00;
border: 2px solid black;
	font: 11pt &quot;Courier New&quot;;
}</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="guessedDisplay">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid black;
	font: 12pt &quot;Courier New&quot;;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="restartBut">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>383</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #000000;
	color: #00FF00;
	border: 2px solid #00FF00;
	font: 12pt &quot;Courier New&quot;;
}

QPushButton::pressed {
	color: #2E7339;
	border: 2px solid #2E7339;
}

QPushButton::disabled {
	color: #2E7339;
	border: 2px solid #2E7339;
}</string>
    </property>
    <property name="text">
     <string>RESTART</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
